src/matrixrtc/CallMembership.ts

```diff
@@ -29,6 +30,87 @@ import { type RTCCallIntent } from "./types.ts";
 export const DEFAULT_EXPIRE_DURATION = 1000 * 60 * 60 * 4;
 
 type CallScope = "m.room" | "m.user";
+type Member = { user_id: string; device_id: string; id: string };
+
+export interface RtcMembershipData {
+    "slot_id": string;
+    "member": Member;
+    "m.relates_to"?: {
+        event_id: string;
+        rel_type: RelationType.Reference;
+    };
+    "application": {
+        type: string;
+        // other application specific keys
+        [key: string]: any;
+    };
+    "rtc_transports": Transport[];
+    "versions": string[];
+    "sticky_key"?: string;
```

Comment:
should be the unstable msc field?

Reply:
in the content? I though in the content we dont use the unstable field?
> The sticky key in the content of the PDU is msc4354_sticky_key. 

It seems you are right.